python_modules/dagster/dagster/_core/definitions/assets_job.py

```diff
@@ -164,6 +169,9 @@ def asset2(asset1):
     source_assets = check.opt_sequence_param(
         source_assets, "source_assets", of_type=(SourceAsset, AssetsDefinition)
     )
+    asset_checks = check.opt_sequence_param(
+        asset_checks, "asset_checks", of_type=AssetChecksDefinition
+    )
```

Comment:
Internally we should use the naming convention `asset_checks_defs`. I know we are already there with `assets` and `AssetsDefinitions` in terms of counterintuitive/deceptive pluralization in variable name. Seeing us introduce another instance of it in our public api makes me physically wince.

Reply:
Ah I see you are using `asset_checks_defs` elsewhere internally.